scss/civicrm/dashboard/_dashboard.scss

```diff
@@ -0,0 +1,57 @@
+$dashboard-dashlets-margin: 10px;
+
+.crm-inactive-dashlet-fieldset {
+  legend {
+    background: none;
+  }
+
+  .crm-hover-button {
+    color: $brand-primary;
+
+    &:hover {
+      color: darken($brand-primary, $crm-darken-percentage);
+    }
+  }
+}
+
+#civicrm-dashboard {
```

Comment:
Cant we remove `#civicrm-dashboard`? that way the styles will be for dashboard styles directly, means h=they can be used in other pages also if needed.

Reply:
Unfortunately the designs would break and we would need to rely on `!important` statements because some of the rules that we are trying to override use `#civicrm-dashboard`.